### PR TITLE
Export CSV for proposal Model in the Admin Panel

### PR DESCRIPTION
**Pull request by AkhilV276**
Added CSV Export option in Admin Panel for Proposal Model.

### CODE REVIEW

**Comment by ananyo2012** [comment]
Sample export
[exported_data.csv](https://github.com/pythonindia/junction/files/12799631/exported_data.csv)

**Comment by ananyo2012** [comment]
image
Exported data looks good

File: junction/proposals/admin.py [hunk 1]
```diff
@@ -32,26 +54,27 @@ def get_queryset(self, request):
...
 
 @admin.register(models.ProposalType)
 class ProposalTypeAdmin(AuditAdmin):
     list_display = (
-        "name",
-        "active",
-        "start_date",
-        "end_date",
-    ) + AuditAdmin.list_display
+                       "name",
```
Comment (ananyo2012) [inline junction/proposals/admin.py:69]: Can you please remove these additional white spaces ? There are some more in the subsequent lines.

**Review by ananyo2012 — CHANGES_REQUESTED** [review]

File: junction/proposals/admin.py (hunk 1 above)
Comment (AkhilV276) [reply junction/proposals/admin.py]: Removed.

**Review by ananyo2012 — APPROVED** [review]

**Pull request merged**